### PR TITLE
Feature/10355 fss UI search screen refinement

### PR DESCRIPTION
**Pull request by pradnya13705**
Please review this PR about screen refinement of landing page and accessibility changes

### CODE REVIEW

File: src/app/features/fss-search/fss-search.component.html [hunk 1]
```diff
@@ -1,9 +1,13 @@
-<div class="spinner-wrapper" *ngIf="displayLoader"><i class="fas fa-spinner fa-pulse fa-3x"></i></div>
+<div class="spinner-wrapper" *ngIf="displayLoader" aria-live='polite' role="alert" aria-label="results loding"><i class="fas fa-spinner fa-pulse fa-3x" aria-hidden="true"></i></div>
```
Comment (martyn-fewtrell) [inline src/app/features/fss-search/fss-search.component.html:1]: Assume this should read aria-label="Results Loading"

Capitalise and correct spelling.

Not sure about the capitalisation but noticed "Download File" is capitalised elsewhere (previous file) so we should be consistent.

**Review by martyn-fewtrell — COMMENTED** [review]
Looking good.

If we can resolve the typo ill get this merged to main!

File: src/app/features/fss-search/fss-search.component.html (hunk 1 above)
Comment (pradnya13705) [reply src/app/features/fss-search/fss-search.component.html]: Resolve Typo as commented .

Thanks

**Review by martyn-fewtrell — APPROVED** [review]
Thanks for the update

**Pull request merged**